src/components/Basic/BorrowTabs/RepayBorrowTab.tsx

```diff
@@ -270,47 +274,69 @@ $TSFixMe) {
                   width: 25,
                   height: 25,
                   marginLeft: 2,
-                  marginRight: 16
+                  marginRight: 16,
                 }}
                 src={vaiImg}
                 alt="asset"
               />
               <span>Repay VAI Balance</span>
             </div>
-            <span>{userVaiMinted.dp(2, 1).toString(10)} VAI</span>
+            <span>
+              {userVaiMinted.dp(2, 1).toString(10)}
```

Comment:
`${userVaiMinted.dp(2, 1).toString(10)} VAI`

Reply:
I think, would be nice to setup prettier rules in task with pre-commit hook. now it's just imported from airbnb config